docs/python/environments.md

```diff
@@ -85,7 +86,7 @@ This will add the path to the Python interpreter from the new virtual environmen
 
 ### Create a conda environment
 
-The Python extension automatically detects existing conda environments provided that the environment contains a Python interpreter. For example, the following command creates a conda environment with the Python 3.4 interpreter and several libraries, which VS Code then shows in the list of available interpreters:
+The Python extension automatically detects existing conda environments. We recommend you install a Python interpreter into your conda environment, otherwise one will be installed for you after you select the environment. For example, the following command creates a conda environment with the Python 3.4 interpreter and several libraries, which VS Code then shows in the list of available interpreters:
 
 ```bash
 conda create -n env-01 python=3.4 scipy=0.15.0 astroid babel
```

Comment:
Also in the additional note section:

> If you create a new conda environment while VS Code is running, use the **Reload Window** command to refresh the environment list shown with **Python: Select Interpreter**; otherwise you may not see the environment there. It might take a short time to appear; if you don't see it at first, wait 15 seconds then try using the command again

Users can now use the refresh icon on the top right of the interpreters list instead of reloading:

![image](https://user-images.githubusercontent.com/13199757/162548522-d02d3950-c785-42c8-8043-37c3b8170704.png)

So we can possibly point to that instead. cc/ @luabud Although that won't detect any new global installs.